plasma_cash/child_chain/child_chain.py

```diff
@@ -10,14 +14,32 @@ def __init__(self, authority=plasma_config['AUTHORITY'],
         self.root_chain = root_chain
         self.authority = authority
         self.blocks = {}
+        self.current_block = Block()
         self.current_block_number = 1
 
         # Register for deposit event listener
         deposit_filter = self.root_chain.on('Deposit')
         deposit_filter.watch(self.apply_deposit)
 
     def apply_deposit(self, event):
-        new_owner = event['args']['depositor']
+        new_owner = utils.normalize_address(event['args']['depositor'])
         amount = event['args']['amount']
         uid = event['args']['uid']
         deposit_tx = Transaction(0, uid, amount, new_owner)
+        self.current_block.transaction_set[uid] = deposit_tx
+
+    def submit_block(self, sig):
+        signature = bytes.fromhex(sig)
+        assert signature != b'\x00' * 65
```

Comment:
Minor: I think we can avoid assert and use proper exception in this package from the beginning